src/query/domain/EntityConfig.ts

```diff
@@ -0,0 +1,20 @@
+export class EntityAttribute {
+  constructor(
+    public name: string,
+    public type: string,
+  ) {}
+}
+
+export class Entity {
```

Comment:
Just a side remark: in frontend we often talk about "entities" for concrete docs in the db, while "EntityType" is the definition with a general label and the fields these type of entities can have.

I think your naming here is simple and nice - but might not map exactly with that use in ndb-core?

Reply:
So you would recommend to rename "Entity" to "EntityType"?